doc/contributors_guide.rst

```diff
@@ -0,0 +1,13 @@
+*******************
+Contributor's guide
+*******************
+
+Handling subtrees
+=================
+
+
+
+Contribution rules
+==================
+
+See `this document <https://github.com/ARM-software/lisa/blob/next/CONTRIBUTING.md>`__.
```

Comment:
not sure if that was already discussed, but github can render reStructuredText when named `.rst` (there are plain reST, no Sphinx extension). That way, we could `.. include::` the source here directly.

Reply:
I gave that a try and Github rendered a very plain textfile (I had no syntax highlight for the python snippet). Maybe I missed something - being able to inline the file would be nice indeed.

Reply:
You can try to start from that source file:
https://github.com/ARM-software/arm-trusted-firmware/blob/master/readme.rst

And then update the content one piece at a time, maybe there is a directive that is a sphinx extension that make it fail, or something like that